src/main/java/io/github/townyadvanced/townyresources/controllers/PlayerExtractionLimitsController.java

```diff
@@ -78,9 +78,9 @@ public static void loadAllResourceExtractionCategories() throws Exception{
      * @param event the event
      */
     public static void processEntityDamageByEntityEvent(EntityDamageByEntityEvent event) {
-        if(!TownyResourcesSettings.areResourceExtractionLimitsEnabled())
+        if(!TownyResourcesSettings.areResourceExtractionLimitsEnabled() || !TownyResourcesSettings.areDropsExtractionLimitsEnabled())
             return;
-    
+
```

Comment:
line 83 and line 106 had white spacing changes made which should be reverted.

Reply:
Formatting resolved in 022cee9

Reply:
Line 83 is still off, line 87 has now been changed.

Reply:
Attempt number 2 haha 8312a3c

Reply:
a1316cc